txclib/commands.py

```diff
@@ -46,39 +45,29 @@ def cmd_init(argv, path_to_tx):
     else:
         path_to_tx = os.getcwd()
 
+    print(messages.init_intro)
     save = options.save
     # if we already have a config file and we are not told to override it
     # in the args we have to ask
-    if os.path.isdir(os.path.join(path_to_tx, ".tx")) and not save:
-        logger.info("There is already a tx folder!")
-        if not utils.confirm(
-            prompt='Do you want to delete it and reinit the project?',
-            default=False
-        ):
-            return
-        # Clean the old settings
-        # FIXME: take a backup
-        else:
-            save = True
-            rm_dir = os.path.join(path_to_tx, ".tx")
-            shutil.rmtree(rm_dir)
+    if os.path.isdir(os.path.join(path_to_tx, ".tx")):
+        if not save:
+            logger.info(messages.init_initialized)
+            if not utils.confirm(messages.init_reinit):
+                return
+        rm_dir = os.path.join(path_to_tx, ".tx")
+        shutil.rmtree(rm_dir)
```

Comment:
@SofiaMargariti this seems a bit too destructive. Could we not just delete the `~/.tx/config` file instead of the whole directory? As of today we don't store anything else in `~/.tx/` I think, but this could change. On line 70 we check for the existence of the file anyway so we could remove or truncate it when `save==True` or the user confirms the action.